packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -7640,6 +7784,12 @@ describe('plugin-meetings', () => {
           meeting.audio = {handleLocalStreamChange: sinon.stub()};
           meeting.video = {handleLocalStreamChange: sinon.stub()};
           meeting.statsAnalyzer = {updateMediaStatus: sinon.stub()};
+          meeting.shareMetricSentStatus = {
+            transmitStart: sinon.stub(),
```

Comment:
why are we setting these to stub() functions and not just `false`? these are not functions, just boolean fields